app/src/main/java/org/p2p/wallet/transaction/ui/ProgressBottomSheet.kt

```diff
@@ -187,4 +155,20 @@ class ProgressBottomSheet : NonDraggableBottomSheetDialogFragment() {
             lineView.isVisible = false
         }
     }
+
+    private fun handleError(state: TransactionState.Error) {
+        with(binding) {
+            titleTextView.text = state.message
+            progressBar.isVisible = false
+
+            transactionImageView.setImageResource(R.drawable.ic_error_transaction)
+            lineView.isVisible = true
+            lineView.setBackgroundColor(getColor(R.color.systemErrorMain))
+        }
+    }
+
+    private fun setResultAndDismiss(result: Bundle = bundleOf(EXTRA_RESULT_KEY_DISMISS to null)) {
```

Comment:
the bundle is always `EXTRA_RESULT_KEY_DISMISS to null`, check your method calls! 
or remove this parameter